.github/workflows/dependabot-auto-merge.yaml

```diff
@@ -19,7 +19,7 @@ jobs:
           github-token: "${{ secrets.GITHUB_TOKEN }}"
       - name: Enable auto-merge for Dependabot PRs
         if: steps.metadata.outputs.update-type == 'version-update:semver-patch' || steps.metadata.outputs.update-type == 'version-update:semver-minor'
-        run: gh pr merge --auto --merge "$PR_URL"
+        run: gh pr merge --auto --squash "$PR_URL" && gh pr review --approve "$PR_URL"
```

Comment:
should this approve first, then squash-and-merge?

Reply:
I don't think it matters - `--auto` will set the PR to auto-merge, is my understanding

From `gh pr merge --help`:
> `--auto                    Automatically merge only after necessary requirements are met`